source/extensions/filters/http/ext_proc/processor_state.cc

```diff
@@ -508,9 +523,10 @@ absl::Status ProcessorState::handleTrailersResponse(const TrailersResponse& resp
         filter_.onProcessTrailersResponse(response, mut_status, trafficDirection());
         return mut_status;
       }
+      processing_effect = ProcessingEffect::Effect::ContentModified;
```

Comment:
Same as the header response case that the header_mutation in the ProcessingResponse may ends up no trailer modification to the original HTTP request at all.